annileen/engine/application.ixx

```diff
@@ -69,9 +67,9 @@ export namespace annileen
 	private:
 		Engine* m_Engine{ nullptr };
 		std::string m_ApplicationName{ "" };
-		Camera* m_NoCamera{ nullptr };
+		std::shared_ptr<Camera> m_NoCamera{ nullptr };
```

Comment:
i think it could be a `std::weak_ptr`, because it's just a "reference" to the camera with no ownership intentions, right?

Reply:
You're right.